src/Token/AccessToken.php

```diff
@@ -34,11 +34,11 @@ class AccessToken
      */
     public function __construct(array $options = null)
     {
-        if (! isset($options['access_token'])) {
-            throw new \InvalidArgumentException(
-                'Required option not passed: access_token'.PHP_EOL
-                .print_r($options, true)
-            );
+        if (empty($options['access_token'])) {
+            throw new InvalidArgumentException(sprintf(
+                'Required option not passed: "%s"',
+                'access_token'
```

Comment:
Actually I'm not sure I want to do that, as it will change the exception type from `InvalidArgument` to `BadMethodCall`. The other usage is for an actual method call, but this is for a constructor... what do you think, @ramsey ?